2021-05-19 16:00:49,352 ERROR [main] [DatabaseManager.java:31] Ошибка при установлении соединения с базой данных
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:32] ВАЖНО: пользователь "postgres" не прошёл проверку подлинности (по паролю) (pgjdbc: autodetected server-encoding to be windows-1251, if the message is not readable, please check database logs and/or host, port, dbname, user, password, pg_hba.conf)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:613)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:161)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:213)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:51)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.Driver.makeConnection(Driver.java:465)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] org.postgresql.Driver.connect(Driver.java:264)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] java.sql.DriverManager.getConnection(DriverManager.java:664)
2021-05-19 16:00:49,355 ERROR [main] [DatabaseManager.java:35] java.sql.DriverManager.getConnection(DriverManager.java:247)
2021-05-19 16:00:49,356 ERROR [main] [DatabaseManager.java:35] server.util.DatabaseManager.establishConnectionWithDatabase(DatabaseManager.java:28)
2021-05-19 16:00:49,356 ERROR [main] [DatabaseManager.java:35] server.util.DatabaseManager.<init>(DatabaseManager.java:20)
2021-05-19 16:00:49,356 ERROR [main] [DatabaseManager.java:35] server.Server.main(Server.java:40)
